### PR TITLE
Add slurm configuration for ReleaseDump pipeline

### PR DESCRIPTION
**Pull request by nakib103**
[ENSVAR-6277](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-6277)

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm [hunk 1]
```diff
@@ -167,9 +167,12 @@ sub resource_classes {
     my ($self) = @_;
     return {
         %{$self->SUPER::resource_classes},
-        'default' => { 'LSF' => '-q production -R"select[mem>1500] rusage[mem=1500]" -M1500'},
-        'medium'  => { 'LSF' => '-q production -R"select[mem>4500] rusage[mem=4500]" -M4500'},
-        'high'    => { 'LSF' => '-q production -R"select[mem>8500] rusage[mem=8500]" -M8500'},
+        'default' => { 'LSF' => '-q production -R"select[mem>1500] rusage[mem=1500]" -M1500',
+                       'SLURM' => '--partition=production --time=10:00:00 --mem=1500M'},
```
Comment (dglemos) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm:171]: You could request more memory, on slurm the jobs need a bit more.

4GB is ok

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm [hunk 2]
```diff
@@ -167,9 +167,12 @@ sub resource_classes {
...
+        'medium'  => { 'LSF' => '-q production -R"select[mem>4500] rusage[mem=4500]" -M4500',
+                       'SLURM' => '--partition=production --time=10:00:00 --mem=4500M'},
```
Comment (dglemos) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm:173]: Here you could request 6 or 8GB

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm [hunk 3]
```diff
@@ -167,9 +167,12 @@ sub resource_classes {
...
+        'high'    => { 'LSF' => '-q production -R"select[mem>8500] rusage[mem=8500]" -M8500',
+                       'SLURM' => '--partition=production --time=10:00:00 --mem=8500M'},
```
Comment (dglemos) [inline modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm:175]: And here 12GB

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm (hunk 1 above)
Comment (nakib103) [reply modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm]: Thanks Diana, added more memory.

File: modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm (hunk 2 above)
Comment (nakib103) [reply modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/ReleaseDumps_conf.pm]: done

**Review by dglemos — APPROVED** [review]

**Review by vinay-ebi — APPROVED** [review]

**Comment by dglemos** [comment]
Merged into release/113 and main

**Pull request merged**